jupyterlite_xeus_python/build.py

```diff
@@ -230,9 +232,14 @@ def build_and_pack_emscripten_env(
 
             worker = worker.replace("XEUS_KERNEL_FILE", "'xpython_wasm.js'")
             worker = worker.replace("LANGUAGE_DATA_FILE", "'python_data.js'")
-
+            worker = worker.replace("importScripts(DATA_FILE);", """
+                importScripts(DATA_FILE);
+                await globalThis.Module.importPackages();
+                await globalThis.Module.init();
+            """ )
```

Comment:
This will not be executed in the case of the `jupyter lite build` CLI, unless we rebuild the `worder.ts` at that stage, which we cannot do.

This may be the source of errors that @davidbrochart has seen

Reply:
We may want to add this logic to xeus-lite directly, @DerThorsten what do you think?